std_msgs/Header header
float64 force_x
float64 force_y
float64 moment_z
